crates/uv-resolver/src/lock/mod.rs

```diff
@@ -71,7 +72,7 @@ mod tree;
 pub const VERSION: u32 = 1;
 
 /// The current revision of the lockfile format.
-const REVISION: u32 = 1;
+const REVISION: u32 = 2;
```

Comment:
\o/

Reply:
Do we need any process around rolling out a new revision?

Reply:
Wait it looks like upload time is optional? Do we need to bump the revision number here?

Reply:
(REVISION != VERSION)

Reply:
Ah okay!